### DISCUSSION TITLE
Frequency dependent line parameters.

### DISCUSSION THREAD

**Discussion by jd-lara** [NREL-SIIP/PowerSimulationsDynamics.jl#86] [General]
Should we implement frequency-dependent line parameters? Will it affect meaningfully the results?

**Comment by rodrigomha** [comment]
I'll say, based on Milano's book, that there is some effect, but it is minor. Nevertheless, implementing for each bus a frequency value (that could come from COI, frequency divider or reference machine), it is a valuable feature that we should implement some time in the future (probably before a 1.0 release).
